mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -134,14 +143,9 @@ def load(context, name, playlist_slice=slice(0, None)):
     - MPD 0.17.1 does not fail if the specified range is outside the playlist,
       in either or both ends.
     """
-    uri = context.lookup_playlist_uri_from_name(name)
-    playlist = uri is not None and context.core.playlists.lookup(uri).get()
-    if not playlist:
-        raise exceptions.MpdNoExistError('No such playlist')
-
-    with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', 'tracklist.add.*"tracks".*')
-        context.core.tracklist.add(playlist.tracks[playlist_slice]).get()
```

Comment:
I don't understand. `track_uris` here are Track Refs. Wasn't the intent here to catch deprecation warnings? Which are not longer being emitted since I'm using the `uris` parameter instead of `tracks`?